spring-cloud-gcp-pubsub/src/main/java/org/springframework/cloud/gcp/pubsub/core/PubSubTemplate.java

```diff
@@ -162,17 +162,33 @@ public void ack(Collection<AcknowledgeablePubsubMessage> acknowledgeablePubsubMe
 		this.pubSubSubscriberTemplate.ack(acknowledgeablePubsubMessages);
 	}
 
+	@Override
+	public void ackAsync(Collection<AcknowledgeablePubsubMessage> acknowledgeablePubsubMessages) {
+		this.pubSubSubscriberTemplate.ack(acknowledgeablePubsubMessages);
+	}
+
 	@Override
 	public void nack(Collection<AcknowledgeablePubsubMessage> acknowledgeablePubsubMessages) {
 		this.pubSubSubscriberTemplate.nack(acknowledgeablePubsubMessages);
 	}
 
+	@Override
+	public void nackAsync(Collection<AcknowledgeablePubsubMessage> acknowledgeablePubsubMessages) {
+		this.pubSubSubscriberTemplate.nack(acknowledgeablePubsubMessages);
```

Comment:
DITTO, although of course I mean `nackAsync()` 😄

Reply:
Nice catch! If we had the async methods return something, this typo could've been avoided. :)